cypress-io/cypress
# chore: update schematic test text to match angular default component …

Update cypress schematic test text to run green on default angular-cli cypress installation



- Closes [Angular-cli "ng e2e" command use cypress @cypress/schematic@2.5.1 but default test fail](https://github.com/cypress-io/cypress/issues/29585)

### Additional details

Just a small improvement to have a default cypress test running green when installing cypress using ng e2e command.
here is the [default component template from angular-cli](https://github.com/angular/angular-cli/blob/8ae395850e4dc874b9bd62321c9dd5d3a1adbca1/packages/schematics/angular/application/files/common-files/src/app/app.component.html.template#L229)

### Steps to test

- create a new angular application with ng new command
- run ng e2e and confirm all steps
- run ng e2e command again to run cypress test and it will pass

### How has the user experience changed?

Default install of cypress with angular cli will create a test that not fail

### PR Tasks

[na]
